src/commands/configure/domain.rb

```diff
@@ -13,7 +13,7 @@ def setup(_args, _options); end
         protected
 
         def answers_file
-          File.join(config.answer_files_path, 'domain.yaml')
+          config.domain_answers_file
```

Comment:
Ok so leave it as is. But I may need to change it when it comes to merging #134. This abstracts away some of the information I need.